test/performance/README.md

```diff
@@ -0,0 +1,73 @@
+# Performance testing the IPS implementation
+
+## Setup details
+
+The performance runner utilizes K6 along with custom extensions for generating fake data.
+
+To include additional extensions, refer to the following resources to build your desired binary:
+
+- [Build a K6 Binary using Docker](https://k6.io/docs/extensions/guides/build-a-k6-binary-using-docker/)
+- [K6 Extensions Documentation](https://k6.io/docs/extensions/)
+
+Faker Extension:
+
+- [GitHub Repository](https://github.com/szkiba/xk6-faker)
+- [Documentation](https://ivan.szkiba.hu/xk6-faker/index.html)
+
+## Building a k6 binary
+
+Example command to build a binary with the Faker extension:
+
+```
+docker run --rm -u "$(id -u):$(id -g)" -v "${PWD}:/xk6" grafana/xk6 build --with github.com/szkiba/xk6-faker --with github.com/grafana/xk6-output-prometheus-remote@latest
+```
+
+## Run a specific performance test
```

Comment:
I'm thinking we should also include a smoke test which runs just 1 transaction and ensures it succeeds. We should suggest in the docs that this is alway run first as some service need to be warmed up before they perform well.